releng_changelog.md

```diff
@@ -6,44 +6,44 @@ This page best serves the people that have previously been into the releaseduty
 
 _**As releaseduty squirrels are the ones with the best context when it comes to releases, they are the ones to edit this page and amend it accordingly. Keep in mind that changes should be in compliance with the other pieces of documentation.**_
 
-## During 82.0 >= 2020-09-21
-### Changed
+### During 82.0 >= 2020-09-21
```

Comment:
I'm hoping to fold the changelog subsection by this. With time, that section will keep growing and it's hard to read the rest of the index because of it.